src/me.rs

```diff
@@ -209,34 +209,33 @@ mod nasm {
 
   #[inline(always)]
   pub fn get_sad<T: Pixel>(
-    plane_org: &PlaneSlice<'_, T>, plane_ref: &PlaneSlice<'_, T>, blk_h: usize,
-    blk_w: usize, bit_depth: usize
+    plane_org: &PlaneSlice<'_, T>, plane_ref: &PlaneSlice<'_, T>, blk_w: usize, blk_h: usize, bit_depth: usize
   ) -> u32 {
     #[cfg(all(target_arch = "x86_64", feature = "nasm"))]
     {
       if mem::size_of::<T>() == 2 && is_x86_feature_detected!("ssse3") && blk_h >= 4 && blk_w >= 4 {
         return unsafe {
           let plane_org = &*(plane_org as *const _ as *const PlaneSlice<'_, u16>);
           let plane_ref = &*(plane_ref as *const _ as *const PlaneSlice<'_, u16>);
-          sad_hbd_ssse3(plane_org, plane_ref, blk_h, blk_w, bit_depth)
+          sad_hbd_ssse3(plane_org, plane_ref, blk_w, blk_h, bit_depth)
```

Comment:
The parameters of `sad_hbd_ssse3` are still in `(h, w)` order.

Reply:
Should I reorder the parameters or the arguments?

Reply:
I think everything should be in (width, height) order.